.env

```diff
@@ -61,6 +61,7 @@ SITE_URL=http://${DEV_DOMAIN:-localhost}:$SITE_PORT
 SITE_PRELOGIN_ENABLED=false
 SITE_PRELOGIN_PASSWORD=password
 NEXT_PUBLIC_SITE_DOMAIN=main
+API_BASIC_AUTH_SYSTEM_USER_PASSWORD=$BASIC_AUTH_SYSTEM_USER_PASSWORD
```

Comment:
can't we just rename BASIC_AUTH_SYSTEM_USER_PASSWORD to API_BASIC_AUTH_SYSTEM_USER_PASSWORD?

Reply:
The API prefix is redundant [here](https://github.com/vivid-planet/comet/blob/44bc70a41c2e13914b07fb95710b0db2a18cda03/demo/api/src/config/environment-variables.ts#L38).

@johnnyomair @thomasdax98 @fraxachun  what do you think?